libs/blocks/gnav/gnav.css

```diff
@@ -197,11 +197,16 @@ header .breadcrumbs li:first-of-type {
   padding: 0;
 }
 
+header .breadcrumbs li:last-of-type {
+  color: #2c2c2c;
```

Comment:
```suggestion
  color: var(--text-color);
```

Reply:
Updated to use variables.
If the last breadcrumb is hidden then there is nothing to style, looks like the li is just empty, you wouldn't see the darker text.